pyTigerGraph/gds/gsql/featurizer/degrees.gsql

```diff
@@ -0,0 +1,8 @@
+CREATE QUERY degrees()  SYNTAX V1{
```

Comment:
Looks like this function is not included in the allowlist. What is it for?

Reply:
Just to check when the algorithm does not have any input parameters

Reply:
got it. but I see there is such a file in the tests folder. Is this one a duplicate?

Reply:
yeah, that should be a duplicate. I created the same gsql folder in test as well, but we do not need that

Reply:
cool. thanks.